src/app/beer_garden/api/http/handlers/v1/command_publishing_blocklist.py

```diff
@@ -101,21 +113,33 @@ def post(self):
         """
         commands = self.schema_validated_body(CommandPublishingBlocklistListInputSchema)
         checked_gardens = []
+        if len(commands["command_publishing_blocklist"]) == 0:
+            raise BadRequest(reason="Empty list was submitted")
         for command in commands["command_publishing_blocklist"]:
+            target_garden = command["namespace"]
+            if command["namespace"] != config.get("garden.name"):
+                target_garden = Garden.objects.get(
+                    namespaces__contains=command["namespace"]
```

Comment:
It is so it can get the child garden's name since using namespace as garden's name results in a routing error for grandchildren because it can't route directly to the grandchild.